be/src/vec/exec/scan/new_jdbc_scanner.cpp

```diff
@@ -163,6 +164,13 @@ Status NewJdbcScanner::_get_block_impl(RuntimeState* state, Block* block, bool*
 Status NewJdbcScanner::close(RuntimeState* state) {
     RETURN_IF_ERROR(VScanner::close(state));
     RETURN_IF_ERROR(_jdbc_connector->close());
+    JdbcConnector::JdbcStatistic& jdbc_statistic = _jdbc_connector->get_jdbc_statistic();
```

Comment:
Better not update counter when close().
Because close() may be called after query is done, so FE will not receive these metrics.

Reply:
done